### DISCUSSION TITLE
[blocks]:  Funtional Gantt Charts

### DISCUSSION THREAD

**Discussion by Daniel-Kenan** [shadcn-ui/ui#4830] [Blocks Request]
### Description

![code_2023-14-14-27-09](https://github.com/user-attachments/assets/f4a067c0-fc89-42c3-8791-833dd4d1469e)
I would appreciate the above gantt charts for the community. Thank you devs and happy coding.

**Comment by RestartDK** [comment]
I agree using maybe recharts could be viable to make something like this
